storybookjs/aem
# Added lerna to support the monorepo

Issue: The `yarn build` command from the root directory was compiling all of the code to the `app/aem` directory, instead of to each package's directory.

## What I did
Added lerna to support the monorepo structure of the codebase. Updated the build command to use lerna to build to each package directory.

## How to test
Run `yarn build` from the root. See that the CLI and App output files to their own dist folders, instead of everything in the app/aem/dist folder

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no
